stbt_rig.py

```diff
@@ -352,6 +354,12 @@ def argparser():
         test-results view in the Stb-tester Portal -- see
         <https://stb-tester.com/manual/user-interface-reference#filter>.''')
 
+    encrypt_secret_parser = subcommands.add_parser(
+        "encrypt-secret", help="Write encrypted value to config file")
+    encrypt_secret_parser.add_argument(
+        "key", help="Key that can be used to retrieve the secret")
```

Comment:
`name` would be a better name than `key`, to avoid confusion with the encryption keys.

Reply:
Good idea, done.